### DISCUSSION TITLE
group account restricted, bot thought we were cheating

### DISCUSSION THREAD

**Discussion by barrettkwise** [codewars/codewars.com#2652] [Q&A] [answered]
My account was restricted with the message: "Account restricted for violation of terms of service". I do these problems at school and work with a few of my friends to solve the problems. We use my account collectively to submit our answers. Therefore, I copy paste our collective answers into Codewars. How can I get our kata back?

**Comment by hobovsky** [comment, marked as answer]
> My account was restricted with the message: "Account restricted for violation of terms of service". I do these problems at school and work with a few of my friends to solve the problems. We use my account collectively to submit our answers. Therefore, I copy paste our collective answers into Codewars. How can I get our kata back?

After reviewing the solutions it turns out that solutions to all purple kata and most of blue kata are copies of other, old solutions. It seems that some of your friends did not play fair and just submitted not their code as solutions for difficult, valuable kata, effectively leading to the account being trapped.

I was not the bot who invalidated the solutions and blocked the account, but if you feel any need of discussing the issue any further, you can reach out to a moderator on `#support-requests` channel of Codewars Discord. However, seeing the copied solutions,  I doubt the case can be helped.

Cheers

**Reply by barrettkwise** [reply]
Thanks for the response!

Do you think there is any hope for my account? I don't know how I could prove that I was the one not cheating. If you have any ideas, please let me know.

**Reply by Blind4Basics** [reply]
see discord
